pandas-stubs/core/frame.pyi

```diff
@@ -160,8 +160,8 @@ class _LocIndexerFrame(_LocIndexer):
         | Callable[[DataFrame], IndexType | MaskType | list[HashableT]]
         | list[HashableT]
         | tuple[
-            IndexType | MaskType | list[HashableT] | Hashable,
-            list[HashableT] | slice | Series[bool] | Callable,
+            Iterable[HashableT] | slice | Hashable,
```

Comment:
I don't think you need `HashableT` for `Iterable` as it is covariant: `Iterable[Hashable]`